resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/KubernetesConfSuite.scala

```diff
@@ -44,6 +44,10 @@ class KubernetesConfSuite extends SparkFunSuite {
     "customEnvKey2" -> "customEnvValue2")
   private val DRIVER_POD = new PodBuilder().build()
   private val EXECUTOR_ID = "executor-id"
+  private val EXECUTOR_ENV_VARS = Map(
+    "spark.executorEnv.1executorEnvVars1/var1" -> "executorEnvVars1",
+    "spark.executorEnv.executorEnvVars2*var2" -> "executorEnvVars2",
+    "spark.executorEnv.executorEnvVars3_var3" -> "executorEnvVars3")
```

Comment:
Let's move this into the test case.

Reply:
BTW, please add another test case like `"spark.executorEnv.4executorEnvVars4/var4" -> "executorEnvVars4"` in order to show clearly that this is only `key`-constraint.

Reply:
@dongjoon-hyun  "spark.executorEnv.4executorEnvVars4/var4" -> "executorEnvVars4"  is the same as "spark.executorEnv.1executorEnvVars1/var1" -> "executorEnvVars1" . BTW, I added two test cases, which I think will prove that this is only key-constraint.